apps/backend/src/modules/contributors/contributor.service.ts

```diff
@@ -106,9 +110,11 @@ class ContributorService implements Service {
 		};
 	}
 
-	public async findAllWithoutPagination(): Promise<ContributorGetAllResponseDto> {
+	public async findAllWithoutPagination(
+		hasHidden?: boolean,
+	): Promise<ContributorGetAllResponseDto> {
```

Comment:
It's may be not clear what boolean parameter is used for outside of this service
```suggestion
	public async findAllWithoutPagination({
		hasHidden,
	}: { hasHidden?: boolean }): Promise<ContributorGetAllResponseDto> {
```
For example, you call  `this.contributorService.findAllWithoutPagination(false));` -- when we look at this call it is hard to guess what `false` may be used for.

Same for `findAllByProjectId`